src/issues/issue-2/david-byrne/AnimatedWords.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+
+import { AnimatedWordContainer, AnimatedTextWrap, TextBox } from "./styled"
+
+const AnimatedWords = ({ forwardedRef, heights, renderTransitions }) => {
```

Comment:
note: react will complain if you call a prop `ref` as that is a reserved word, so I named it `forwardedRef`